package.json

```diff
@@ -94,7 +94,7 @@
     "run-sequence": "^1.1.4",
     "sinon": "git://github.com/cjohansen/Sinon.JS#b672042043517b9f84e14ed0fb8265126168778a",
     "sinon-chai": "^2.8.0",
-    "ta-scripts": "^2.3.0",
+    "ta-scripts": "^2.3.1",
```

Comment:
Yes sir, did this strictly for CircleCI which _I think_ would've ran with old cache.  Not 100%.